docker/server/src/main/server-tensorflow/requirements.txt

```diff
@@ -10,10 +10,11 @@ gast==0.4.0
 google-auth==2.8.0
 google-auth-oauthlib==0.4.6
 google-pasta==0.2.0
-grpcio==1.46.3
+grpcio==1.47.0
```

Comment:
why advance this past the 1.46 that the server (and java client) presently uses?

Reply:
that said, make sure all py modules get updated accordingly - at a glance i see sphinx (for our docs) is still using 1.46.0, not changed in this diff

Reply:
This is a requirement inherited from tensorflow - the dependencies regularly get updated. As part of update adding `jpy` as a proper base image dependency, some of the transitive dependencies got updated as well. https://github.com/deephaven/deephaven-base-images/pull/29